src/TypeScriptGenerator.ts

```diff
@@ -1111,7 +1134,9 @@ function getEnumDefinitions(
   }
 
   return enumNames.map(name => {
-    const values = schema.getEnumValues(usedEnums[name]);
+    const values = [].concat(
+      schema.getEnumValues(usedEnums[name]) as any
+    ) as any;
```

Comment:
Why?

Reply:
Concat is waiting a Array<never>. I fixed it by switching to spread in array